Orange/classification/tree.py

```diff
@@ -1,20 +1,254 @@
+"""Tree inducers: SKL and Orange's own inducer"""
+
+import numpy as np
+import bottleneck as bn
 import sklearn.tree as skl_tree
+from Orange.statistics import distribution, contingency
 
-from Orange.base import Tree
+from Orange.base import Tree, Model, Learner
```

Comment:
This Learner will handle both, classification and regression. I've put it to folder `classification` for lack of a better place.